app/api/templates/specs/v2_newRelationshipProperties.spec.ts

```diff
@@ -362,7 +366,7 @@ describe('template.save()', () => {
         ]);
       });
 
-      it('on query change, uwazi should save the query properly, and mark the metadata obsolete', async () => {
+      it('on query change, uwazi should save the query properly, mark the metadata obsolete and recalculate the target templates', async () => {
```

Comment:
We can do so. I didn't because the calculation of the `targetTemplates` value is tested in other places, and was only aiming to check that the calculation is performed.